src/actions/nodes.js

```diff
@@ -19,7 +19,7 @@ export const TOGGLE_NODES_DISABLED = 'TOGGLE_NODES_DISABLED';
 /**
  * Toggle a selected group of nodes' visibility on/off
  * @param {Array} nodeIDs The nodes' unique identifiers
- * @param {Boolean} isDisabled Whether the node should be visible
+ * @param {boolean} isDisabled Whether the node should be visible
```

Comment:
How do you determine whether `boolean` should be capitalized or not, or does it not matter? I made this consistent with the rest of the files, but it's not always consistent with other files.

Reply:
It should probably be capitalized everywhere: https://developer.mozilla.org/en-US/docs/Glossary/Boolean

Reply:
I did see that, but then `String`, etc. should all be capitalized, too. Guess can do it later.